server/routes/generic/fields.ts

```diff
@@ -487,6 +488,30 @@ const fields = {
       },
     ],
   },
+  whereOutsideHospital: {
```

Comment:
as suggested previously, could we leave demos of conditional functionality to purely a single comment box or a single date input?